### ISSUE TITLE
Bower install throws ENORESTARGET error

### ISSUE THREAD

**Issue by jhumigas** [sahat/satellizer#873]
Hello,

I was trying to use this module in my application but I failed to install it with bower.
I tried : `bower install satellizer --save`
Then I got this 
```
bower satellizer#*              cached https://github.com/sahat/satellizer.git#0.15.0
bower satellizer#*             validate 0.15.0 against https://github.com/sahat/satellizer.git#*
bower angular#>=1.3.x && <= 1.5.x       not-cached https://github.com/angular/bower-angular.git#>=1.3.x && <= 1.5.x
bower angular#>=1.3.x && <= 1.5.x          resolve https://github.com/angular/bower-angular.git#>=1.3.x && <= 1.5.x
bower angular#>=1.3.x && <= 1.5.x     ENORESTARGET Tag/branch >=1.3.x && <= 1.5.x does not exist
```

Any way to resolve this ?

Additional information : 

- Node version : 4.4.7
- Npm version : 3.8.3
- Bower version : 1.7.9

**Comment by kkaabbaa** [comment]
the same problem

`Available branches: master, unstable`

**Comment by TablesawTablesawsen** [comment]
This looks like an issue with the semver in bower.json in the latest update. It should be `1.3.x - 1.5.x` or `>=1.3.x <= 1.5.x`, not `>=1.3.x && <= 1.5.x`. I'm using `bower install satellizer#0.14.1` for now.

**Comment by sahat** [comment]
Fixed in #875.
